extension/nine-patch/src/commonMain/kotlin/com/seiko/imageloader/model/NinePatchData.kt

```diff
@@ -0,0 +1,79 @@
+package com.seiko.imageloader.model
+
+import androidx.compose.ui.graphics.FilterQuality
+
+data class NinePatchData internal constructor(
+    val left: Int,
+    val top: Int,
+    val right: Int,
+    val bottom: Int,
+    val scale: Float,
+    val skipPadding: Int,
+    val maxFactor: Float,
+    val filterQuality: FilterQuality,
+)
+
+class NinePatchDataBuilder internal constructor() {
+
+    var left: Int = 0
+        set(value) {
+            check(value >= 0) { "left must >= 0" }
```

Comment:
One of the possibilities to control the value of a number greater than 0 is to use the "UInt" data type; you need to weigh the pros and cons of whether it is advisable to replace the type in NinePatchDataBuilder.

Reply:
I think it would be better to stay with `Int` and avoid the `Int`↔️`UInt` conversion, because the draw side is still using `Int`/`Float`.